go.mod

```diff
@@ -10,8 +10,8 @@ require (
 	github.com/rancher/wrangler v0.7.3-0.20201020003736-e86bc912dfac
 	github.com/rancher/wrangler-api v0.6.1-0.20200427172631-a7c2f09b783e
 	github.com/sirupsen/logrus v1.6.0
-	golang.org/x/oauth2 v0.0.0-20201208152858-08078c50e5b5
-	google.golang.org/api v0.40.0
+	golang.org/x/oauth2 v0.0.0-20210313182246-cd4f82c27b84
+	google.golang.org/api v0.43.0
```

Comment:
Curious why oauth needed to be updated. Also does our GCP client already include this feature ?

Reply:
> Curious why oauth needed to be updated.

It's just what happened when the google api library was updated

> Also does our GCP client already include this feature

No it didn't, it's only been newly added in the last few weeks